content/hi/docs/tutorials/kubernetes-basics/explore/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: ऐप का अन्वेषण करें
```

Comment:
Can you use "Exploring applications" or "Application exploration" to localise in hindi?